applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -55,7 +55,9 @@ static struct slm_httpc_ctx {
 	char *method_str;		/* request method */
 	char *resource;			/* resource */
 	char *headers;			/* headers */
-	size_t pl_len;			/* payload length */
+	char *content_type;		/* Content-Type of payload */
+	size_t content_length;		/* Content-Length of payload, max 65535 */
+	bool trunked_transfer;		/* Trunked transfer or not */
```

Comment:
It's more of a question for tech writer, but is "trunked" the right term here? Wouldn't `chunked` be more appropriate?

Reply:
You're much correct 😅